src/main/resources/db/changelog/changes/01-create-book-table.yaml

```
@@ -0,0 +1,48 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-books-table
+      author: hanna
+      changes:
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

`nullable false` is already included in `primaryKey`
```suggestion
                    primaryKey: true
```